main/test/flix/Test.Type.Bool.flix

```diff
@@ -327,5 +387,6 @@ mod Test.Type.Bool {
     def bitwiseAnd(_: I2[a1, b1], _: I2[a2, b2]): I2[a1 and a2, b1 and b2] = I2.I2
     def swap(_: I2[a, b]): I2[b, a] = I2.I2
     def increment(_: I2[a, b]): I2[((a and (not b)) or (b and (not a))), (not b)] = I2.I2
+    def add(_: I2[a1, b1], _: I2[a2, b2]): I2[((((a1 and (not a2)) or (a2 and (not a1))) and (not (b1 and b2))) or (((a1 and a2) or ((not a1) and (not a2))) and (b1 and b2))), ((b1 and (not b2)) or (b2 and (not b1)))] = I2.I2
```

Comment:
Give this is so complicated, maybe we should write it down like:

```
    def add(_: I2[a1, b1], _: I2[a2, b2]): 
       I2[((((a1 and (not a2)) or (a2 and (not a1))) and (not (b1 and b2))) or (((a1 and a2) or ((not a1) and (not a2))) and (b1 and b2))), 
       ((b1 and (not b2)) or (b2 and (not b1)))] = I2.I2
```

or similar.